src/epsilon/analyzer.d

```diff
@@ -3,6 +3,7 @@ module epsilon.analyzer;
 import EAG = epsilon.eag;
 import Earley = epsilon.earley;
 import epsilon.lexer : Lexer, Token;
+import epsilon.settings : Settings;
```

Comment:
unused import?

Reply:
Yes, your are right.
I first passed the Settings "object" as second parameter to the analyzer.Analyse() method and later restricted it to the offset option value. Then forgot to remove the import.